### DISCUSSION TITLE
Can I add k3s workers/agents into RKE2 cluster

### DISCUSSION THREAD

**Discussion by mbehrisch** [rancher/rke2#4124] [Q&A] [answered]
Hi, 

I have some small ARM-based boards (Jetson Nanos mainly) and would like to add them to an existing rke2 cluster (`Provider: RKE2 Kubernetes Version: v1.24.11`). I am aware of the ARM support issue (https://github.com/rancher/rke2/issues/817), but I don't see why this is not doable right now if the Raspberry PI or Nanos can run k3s and in the end, it is a Kubernetes API question if someone/something can join the cluster.

Best,
Michael

**Comment by brandond** [comment, marked as answer]
No, we do not currently build rke2 for arm or arm64, and you cannot join k3s nodes to an rke2 cluster.

**Reply by mbehrisch** [reply]
Thanks for the reply.
